infrastructure/src/main/java/org/corfudb/infrastructure/management/failuredetector/FailureDetectorService.java

```diff
@@ -145,23 +145,13 @@ public CompletableFuture<DetectorTask> runFailureDetectorTask(PollReport pollRep
     private static class SequencerNotReadyCounter {
         private final long epoch;
         private int counter;
-
-        public void increment() {
-            counter += 1;
-        }
     }
 
     @Builder
     public static class SequencerBootstrapper {
         private static final CompletableFuture<DetectorTask> DETECTOR_TASK_SKIPPED
                 = CompletableFuture.completedFuture(DetectorTask.SKIPPED);
 
-        /**
-         * The management agent attempts to bootstrap a NOT_READY sequencer if the
-         * sequencerNotReadyCounter counter exceeds this value.
-         */
-        private final int sequencerNotReadyThreshold = 3;
```

Comment:
What was the original reasoning to keep this value at 3?

Reply:
LocalMonotoringService worked in parallel with RemoteMonitoringService, and the algorithm was - make sure (check 3 times) that the failure is persistent. Now we trigger sequencer bootstrap from RMS, no need for extra checks, actually extra checks bring huge latency - 1 sequencer bootstrap happens per 2 FD iterations, which is not acceptable